### PR TITLE
updated the transform.py documentation

### PR DESCRIPTION
**Pull request by fmamashli**
#### What does this implement/fix?
There was not clear documentation for the transform Class "from" "to". I got the types of "fro" and "to" from the _verbose_frames dictionary and updated the error message to be more clear.

### CODE REVIEW

File: mne/transforms.py [hunk 1]
```diff
@@ -77,8 +79,18 @@ class Transform(dict):
...
+        'CTF MEG device', 'CTF/4D/KIT head', 'RAS (non-zero origin)',
+        'MNI Talairach', 'Talairach (MNI z > 0)',
+        'Talairach (MNI z < 0)'
     to : str | int
         The ending coordinate frame.
+        Can be 'unknown', 'MEG device', 'isotrak', 'hpi', 'head',
+        'MRI (surface RAS)', 'MRI voxel', 'MRI slice', 'MRI display',
+        'CTF MEG device', 'CTF/4D/KIT head', 'RAS (non-zero origin)',
+        'MNI Talairach', 'Talairach (MNI z > 0)',
+        'Talairach (MNI z < 0)'
```
Comment (larsoner) [inline mne/transforms.py]: It might be better to make a `Notes` section below and list them once down there

File: mne/transforms.py [hunk 2]
```diff
@@ -77,8 +79,18 @@ class Transform(dict):
     ----------
     fro : str | int
         The starting coordinate frame.
+        Can be 'unknown', 'MEG device', 'isotrak', 'hpi', 'head',
```
Comment (jasmainak) [inline mne/transforms.py]: actually we realized this is incorrect. @fmamashli remember to fix these. It should be `'meg'` not `'MEG device'` and so on

File: mne/transforms.py (hunk 1 above)
Comment (agramfort) [reply mne/transforms.py]: +1 to add it in notes

something like "See notes for valid coordinate systems"

```
Notes
------
Valid coordinate systems are ...
```

**Comment by jasmainak** [comment]
@fmamashli can you finish this PR during the sprint?

**Comment by fmamashli** [comment]
Ok :)

**Comment by fmamashli** [comment]
@jasmainak 
I just made a pull request which address this documentation issue ....

**Comment by jasmainak** [comment]
Superceded by #6887

**Pull request closed without merging**